docs/readme.rst

```diff
@@ -1,51 +1,33 @@
-| |Tests|
-| |Version|
-| |License|
+| |Tests| |Version| |License|
 
 Introduction to PyDP
 ====
 
-| In today's data-driven world, more and more researchers and data
-scientists
-| use machine learning to create better models or more innovative
-solutions for
-| a better future.
-
-| These models often tend to handle sensitive or personal data, which
-can cause
-| privacy issues. For example, some AI models can memorize details about
-| the data they've been trained on and could potentially leak these
-details later
-| on.
-
-| To help measure sensitive data leakage and reduce the possibility of
-it
-| happening, there is a mathematical framework called differential
+In today's data-driven world, more and more researchers and data
+scientists use machine learning to create better models or more innovative
+solutions for a better future.
+
+These models often tend to handle sensitive or personal data, which
+can cause privacy issues. For example, some AI models can memorize details about the data they've been trained on and could potentially leak these
+details later on.
+
+To help measure sensitive data leakage and reduce the possibility of
+it happening, there is a mathematical framework called differential
 privacy.
 
-| In 2020, OpenMined created a Python wrapper for Google's `Differential
-Privacy <https://github.com/google/differential-privacy>`__ project
-called PyDP.
-| The library provides a set of ε-differentially private algorithms,
-which can be
-| used to produce aggregate statistics over numeric data sets containing
-private
-| or sensitive information. Therefore, with PyDP you can control the
-privacy
-| guarantee and accuracy of your model written in Python.
+In 2020, OpenMined created a Python wrapper for Google's `Differential
+Privacy <https://github.com/google/differential-privacy>`_ project
+called PyDP. The library provides a set of ε-differentially private algorithms,
+which can be used to produce aggregate statistics over numeric data sets containing
+private or sensitive information. Therefore, with PyDP you can control the
+privacy guarantee and accuracy of your model written in Python.
 
 **Things to remember about PyDP:**
 
--  :rocket: Features differentially private algorithms including:
-   BoundedMean,
-   BoundedSum, Max, Count Above, Percentile, Min, Median, etc.
--  All the computation methods mentioned above use Laplace noise only
-   (other
-   noise mechanisms will be added soon! :smiley:)
--  :fire: Currently supports Linux and macOS (Windows support coming
-   soon
-   :smiley:)
--  :star: Use Python 3.x.
+-  ::rocket: Features differentially private algorithms including: BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median, etc.
+  -  All the computation methods mentioned above use Laplace noise only (other noise mechanisms will be added soon! :smiley:)
```

Comment:
Seems to be indentation issue.

Reply:
I intentionally indented it in my updated README PR: https://github.com/OpenMined/PyDP/pull/279 @chinmayshah99 The Laplace noise applies to the algorithms only, based on the research that I read. Therefore, this can follow a structure similar to:

```
- Feature 1
  - Another feature that applies to Feature 1
- Feature 2
...
```

WDYT?